tests/DependencyInjection/NelmioSecurityExtensionTest.php

```diff
@@ -22,10 +22,15 @@
 use Nelmio\SecurityBundle\ExternalRedirect\AllowListBasedTargetValidator;
 use Nelmio\SecurityBundle\Signer;
 use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\PhpUnit\ExpectDeprecationTrait;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 
 final class NelmioSecurityExtensionTest extends TestCase
 {
+    use ExpectDeprecationTrait {
+        ExpectDeprecationTrait::expectDeprecation as bridgeExpectDeprecation;
```

Comment:
PHPstan (incorrectly) flags the `expectDeprecation` method from the trait as deprecated because `TestCase::expectDeprecation` is deprecated. Renaming it works around this.